firebase/utils/userUtils.js

```diff
@@ -0,0 +1,22 @@
+import { collection, doc, getDoc, serverTimestamp, setDoc, updateDoc } from "firebase/firestore";
+
+import { firestore } from "@/firebase/firebase";
+
+export const userCollection = collection(firestore, "users");
+
+export const createUser = async (uid, data) => {
+  const userRef = doc(userCollection, uid);
+  await setDoc(userRef, { ...data, isOnboarded: false, createdAt: serverTimestamp() });
+};
+
+export const updateUser = async (uid, data) => {
+  const userRef = doc(userCollection, uid);
+  await setDoc(userRef, data, { merge: true });
+};
+
+export const isUserExists = async (uid) => {
```

Comment:
same wording as blog ^

Reply:
// check if user exists by id
export const isExistingUser = async (id) => {
  const userRef = doc(userCollection, id);
  const user = await getDoc(userRef);

  return user.exists();
};

function renamed to 'isExistingUser'